grafana-plugin/src/components/GForm/GForm.tsx

```diff
@@ -193,6 +194,14 @@ class GForm extends React.Component<GFormProps, {}> {
               return null; // don't render the field
             }
 
+            if (formItem.type === FormItemType.PlainLabel) {
```

Comment:
let's maybe use `FormItemType.Other` and provide custom renderer which render just `formItem.label` and nothing else

Reply:
I think we don't need to extend `FormItemType` with  `FormItemType.PlainLabel`, we can make do with existing GForm features

Reply:
it's overhead really, adds just too much code. We will get rid of these anyway, the forms don't work as expected given that there's too many abstraction layers.